### PR TITLE
Add basic-space-cleanup command

### PR DESCRIPTION
**Pull request by tituspijean**
Addresses parts of https://github.com/YunoHost/issues/issues/2230

## How to test

`yunohost tools basic-space-cleanup`

### CODE REVIEW

File: src/tools.py [hunk 1]
```diff
@@ -623,6 +623,36 @@ def tools_shell(command=None):
...
+    archived logs removal
+    journalctl vacuum (leaves 50M of logs)
+    """
+    subprocess.run(
+        [
+            "/bin/bash",
+            "-c",
+            "apt autoremove && apt autoclean",
+        ]
+    )
```
[the remark this answers is not in the document]
Comment (tituspijean) [reply src/tools.py:641]: It is solely due to a complete lack of RTFM. 😅

**Comment by olberger** [comment]
I fear this command lacks documentation, elsewhere that in ```
# yunohost tools basic-space-cleanup --help
usage: yunohost tools basic-space-cleanup [-h]

Basic space cleanup (apt, journalctl, logs, ...)
```
Too bad.

**Comment by alexAubin** [comment]
... which place are you thinking of that it should be documented it ?

**Comment by olberger** [comment]
https://yunohost.org/fr/troubleshooting for a start ? I guess this is pretty empty so far, but never too late to start ?

**Comment by tituspijean** [comment]
You are most welcome to contribute with such a piece of documentation. 🙂

I have recently experienced an issue with a completely full disk, and the command did not help. The issue is that since it is a `yunohost` command, it triggers by default some logging. But it cannot log anything if the disk is full. 🙃

So, before integrating the new paragraph in the troubleshooting documentation, we need to disable logging for that command.

**Pull request merged**